Binary files /dev/null and b/node_modules/react-native-vision-camera/android/build/tmp/kotlin-classes/release/com/mrousavy/camera/utils/WithPromiseKt.class differ
diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraDevicesManager.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraDevicesManager.kt
index 4434ece..27140d7 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraDevicesManager.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraDevicesManager.kt
@@ -71,7 +71,7 @@ class CameraDevicesManager(private val reactContext: ReactApplicationContext) :
     eventEmitter.emit("CameraDevicesChanged", getDevicesJson())
   }

-  override fun hasConstants(): Boolean = true
+  fun hasConstants(): Boolean = true

   override fun getConstants(): MutableMap<String, Any?> {
     val devices = getDevicesJson()
diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/VisionCameraProxy.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/VisionCameraProxy.kt
index 0f8acec..a32a3f1 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/VisionCameraProxy.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/VisionCameraProxy.kt
@@ -35,7 +35,7 @@ class VisionCameraProxy(context: ReactApplicationContext) {

   init {
     val jsCallInvokerHolder = context.catalystInstance.jsCallInvokerHolder as CallInvokerHolderImpl
-    val jsRuntimeHolder = context.javaScriptContextHolder.get()
+    val jsRuntimeHolder = context.javaScriptContextHolder!!.get()
     mScheduler = VisionCameraScheduler()
     mContext = WeakReference(context)
     mHybridData = initHybrid(jsRuntimeHolder, jsCallInvokerHolder, mScheduler)
